solutions/devsprint-bruno-faganello-1/FinanceApp/Network/Endpoint.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Endpoint.swift
+//  FinanceApp
+//
+//  Created by Victor Catão on 17/03/22.
+//
+
+protocol Endpoint {
+    var baseURL: String { get }
+    var path: String { get }
+    var method: RequestMethod { get }
+    var header: [String: String]? { get }
+    var body: [String: String]? { get }
```

Comment:
[Suggestion]
Probably Body and Header in most cases will be null, add a default value in extension